Module4.3_Search_and_Sorting_Algorithms/activities/Act1_SearchingAndSorting/10.md

```diff
@@ -0,0 +1,29 @@
+<!--title={Matchsticks to Square: DFS Algorithm Base Cases}-->
+
+Now that we have established our variables and base cases, let's move on to coding our approach:
+
+**Depth First Search (DFS) Approach**:
+
+It is possible that any given matchstick can be a part of any of the four sides of the resulting square, but choosing which matchstick goes to which side of the resulting square is something we need to figure out.
+
+This means that for each matchstick in our given array, we will have four different options each, representing the side of the square or subset that this matchstick could be a part of.
```

Comment:
can you give an example to better illustrate this point? like for example a matchstick of length 3....